multimodal/vl2l/src/mlperf_inference_multimodal_vl2l/cli.py

```diff
@@ -212,6 +288,7 @@ class Endpoint(BaseModel):
 def main(
     *,
     settings: TestSettings,
+    output_settings: LogOutputSettings,
```

Comment:
If `output_settings` is a separate parameter (i.e., CLI flag group), then the previous `settings` parameter should be named `test_settings`. Or, I wonder if it's cleaner to have a top level `BaseModel` called `Settings`:

```
class Settings(BaseModel):
    test: Annotated[TestSettings, ...]
    log: Annotated[LogSettings, ...]
```
This way `main` only need to take in a single `settings: Settings` parameter. In this case, the CLI flag would look like `--settings.test.min_query_count` and `--settings.log.output.dir`).
I'm okay with either approach.